livre/eq/Config.cpp

```diff
@@ -94,14 +100,27 @@ class Config::Impl
         , eventMapper( EventHandlerFactoryPtr( new EqEventHandlerFactory ))
         , redraw( true )
         , dataFrameRange( INVALID_FRAME_RANGE )
+        , dataSourceRange( std::numeric_limits<uint8_t>::min(), // This will change with the new TF
+                           std::numeric_limits<uint8_t>::max( ))
```

Comment:
You can also add that this range assumes that data sources are 8 bit by default.